turnpike/plugins/x509.py

```diff
@@ -37,21 +37,23 @@ def psk_check(self):
 
     def process(self, context, backend_auth):
         self.app.logger.debug("Begin X509 plugin processing")
-        if self.app.config["AUTH_DEBUG"]:
-            self.app.logger.info(
-                "x.509 headers found: "
-                f"subject={request.headers.get(self.subject_header)} "
-                f"issuer={request.headers.get(self.issuer_header)} "
-                f"psk_ok={self.psk_check()} "
-            )
-        if "x509" in backend_auth and self.subject_header in request.headers and self.psk_check():
+        if "x509" in backend_auth and self.subject_header in request.headers:
+            if not self.psk_check():
+                context.result += "CDN PSK was missing or invalid. "
+                return context
             auth_data = dict(
                 subject_dn=request.headers[self.subject_header], issuer_dn=request.headers.get(self.issuer_header)
             )
-            self.app.logger.debug(f"X509 auth_data: {auth_data}")
             context.auth = dict(auth_data=auth_data, auth_plugin=self)
+            context.log_data.update(
+                {
+                    "x509.issuer.distinguished_name": auth_data["issuer_dn"],
+                    "x509.subject.distinguished_name": auth_data["subject_dn"],
```

Comment:
Same here with regard to logging?

Reply:
These values shouldn't contain PII - they're RHCS issued certs.